migration/20170713-13-move-authentication-configuration-to-external-integrations.py

```diff
@@ -109,8 +109,8 @@ def convert_clever(_db, integration, provider):
     # No need to import configuration if there isn't any.
     sys.exit()
 
+_db = production_session()
 try:
-    _db = production_session()
```

Comment:
An unrelated change that stops this migration script from crashing awkwardly if it's going to crash.